pages/platform/kubernetes-k8s/installing-go-operator/guide.en-gb.md

```diff
@@ -0,0 +1,1390 @@
+The operator allows you to:
+
+ - install a Nginx server with the required number of Pods
+ - upgrade the number of Pods
+ - change the HTTP port
+ - recreate the service if it is deleted
+
+You'll develop this operator with the [operator SDK](https://sdk.operatorframework.io). <br>
+The operator SDK provides several tools:
+
+ - a [CLI](https://sdk.operatorframework.io/docs/cli/) to develop and run locally the developed operator
+ - several helpers in different languages (Helm, Ansible and Go) to easily develop an operator
+
+In this article, you will use the [Go helper](https://sdk.operatorframework.io/docs/building-operators/golang/quickstart/).  
+
+### Install the CLI
+
+The SDK includes a [CLI](https://sdk.operatorframework.io/docs/cli/) (**C**ommand **L**ine **I**nterface).<br>
+In order to install the CLI, follow the instructions applicable to your OS. <br>
+You can, for example, install it through Homebrew:
+
+```bash
+brew install operator-sdk
+```
+
+Then test if the CLI is correctly installed on your computer:
+
+```bash
+operator-sdk version
+```
+
+Output should be like this:
+
+<pre class="console"><code>$ brew install operator-sdk
+...
+==> Installing dependencies for operator-sdk: go
+==> Installing operator-sdk dependency: go
+==> Pouring go--1.17.6.x86_64_linux.bottle.tar.gz
+🍺  /home/linuxbrew/.linuxbrew/Cellar/go/1.17.6: 10,822 files, 532.9MB
+==> Installing operator-sdk
+==> Pouring operator-sdk--1.17.0.x86_64_linux.bottle.tar.gz
+==> Caveats
+Bash completion has been installed to:
+  /home/linuxbrew/.linuxbrew/etc/bash_completion.d
+==> Summary
+🍺  /home/linuxbrew/.linuxbrew/Cellar/operator-sdk/1.17.0: 10 files, 196.3MB
+==> Running `brew cleanup operator-sdk`...
+Disable this behaviour by setting HOMEBREW_NO_INSTALL_CLEANUP.
+Hide these hints with HOMEBREW_NO_ENV_HINTS (see `man brew`).
+==> Caveats
+==> operator-sdk
+...
+
+$ operator-sdk version
+operator-sdk version: "v1.17.0", commit: "704b02a9ba86e85f43edb1b20457859e9eedc6e6", kubernetes version: "v1.21", go version: "go1.17.6", GOOS: "darwin", GOARCH: "arm64"
+</code></pre>
+
+### Develop an operator with Go
+
+In this guide, you will use [Go](https://go.dev/) to create your first operator.<br>
+
+#### Scaffold the project
+
+The CLI offers to scaffold an entire project, you don't use all these files for the demo but it helps you to easily bootstrap the project. <br>
+More information on the project layout generated by the CLI can be found in the [official documentation](https://sdk.operatorframework.io/docs/overview/project-layout/).
+
+First, create the root folder `nginx-go-operator` of your project:
+
+```bash
+mkdir nginx-go-operator
+```
+
+Then in the `nginx-go-operator` folder scaffold the project with the _Operator SDK_:
+
+```bash
+cd nginx-go-operator
+operator-sdk init --project-name nginx-go-operator --domain ovhcloud.com --repo github.com/[your organisation or github account]/nginx-go-operator
+```
+
```

Comment:
Suggestion: add a link to the doc (https://sdk.operatorframework.io/docs/building-operators/golang/tutorial/) and / or explain what parameters values mean.
example : 
--domain ovhcloud.com is used as the registry Docker Hub namespace
--repo=<path> flag is required when creating a project outside of $GOPATH/src